src/platform-implementation-js/dom-driver/gmail/gmail-element-getter.ts

```diff
@@ -175,14 +172,6 @@ const GmailElementGetter = {
 
     // leftNavElement classnames depending on gmail chat & meet settings:
 
-    // Old Hangouts-only Gmail
```

Comment:
<img width="651" alt="Screenshot 2023-08-16 at 13 25 06" src="https://github.com/InboxSDK/InboxSDK/assets/5156873/39c6c748-202a-4ea6-9ae1-6cbbbaec1ee3">

It's my understanding that hangouts support in Gmail is no more. Maybe Chat holds onto these older class names?